js/default-configs/layouts-config.js

```diff
@@ -365,6 +365,13 @@ window.flListLayoutConfig = {
         location: 'Image',
         type: 'image',
         column: 'Image'
+      },
+      {
+        id: 'RGosNfos',
+        interfaceName: 'Secondary text',
```

Comment:
@OleksandrLavrynenko I don't think `interfaceName` is needed. Can you confirm if or why it's necessary?

Reply:
@tonytlwu You're right. This parameter is not needed